Adiciona tradução para add-a-manifest-file.md (#91)
gatsbyjs/gatsby-pt-BR: docs/docs/add-a-manifest-file.md

---

> Título: Adicionando o Arquivo _Manifest

Reviewer: ```suggestion
Título: Adicionando o Arquivo _Manifest_
```

> Título: Adicionando o Arquivo _Manifest_

Reviewer 2: Acho que assim faz mais sentido, oq vc acha?
```suggestion
Título: Adicionando o Arquivo de Manifesto
```

---

Se você executou uma [auditoria com o Lighthouse](/docs/audit-with-lighthouse/), pode ter notado uma pontuação ruim na categoria "Progressive Web App". Vamos abordar como você pode melhorar essa pontuação.

Mas primeiro, o que _são_ exatamente PWAs?

> São sites como qualquer outro porém que aproveitam as funcionalidades dos navegadores mais modernos para melhorar a experiência na _web_ com recursos e benefícios semelhantes a dos aplicativos. Confira o [resumo do Google] sobre esse termo. (https://developers.google.com/web/progressive-web-apps/) para saber o que define uma experiência de PWA e [Progressive web apps (PWAs) doc](/docs/progressive-web-app/) para saber como um site Gatsby é um progressive web app.

Reviewer 2: Aqui o link deu uma quebrada, pois separou o link do alias

`Formato de link esperado: [resumo do Google](link)`

Author: Corrigido @gabrielferreiraa

> A inclusão de um manifesto de aplicativo da web é um dos três, geralmente aceitos, [requisitos mínimos para um PWA](https://alistapart.com/article/yes-that-web-project-should-be-a-pwa#section1).

Reviewer 3: ```suggestion
A inclusão de um manifesto de aplicativo da web é um dos três requisitos mínimos geralmente aceitos para criar um [PWA](https://alistapart.com/article/yes-that-web-project-should-be-a-pwa#section1).
```

> Citação [Google](https://developers.google.com/web/fundamentals/web-app-manifest/):

Reviewer 3: ```suggestion
Citando o [Google](https://developers.google.com/web/fundamentals/web-app-manifest/):
```

> > O manifesto do aplicativo da web é um arquivo JSON simples que informa ao navegador sobre seu aplicativo web e como ele deve se comportar quando 'instalado' no dispositivo móvel ou na área de trabalho do usuário.

Reviewer 3: ```suggestion
> O manifesto do aplicativo da web é um arquivo JSON simples que informa a navegador sobre o seu aplicativo _web_ e como ele deve se comportar quando 'instalado' no dispositivo móvel ou no _desktop_ do usuário.
```

Author: @jay-jlm Segui sua sugestão, apenas deixei o trecho "informa ao navegador" igual, pois você tinha sugerido retirar o `ao` e deixar apenas o `a`.

[Gatsby's manifest plugin](/packages/gatsby-plugin-manifest/) configura o Gatsby para criar um arquivo `manifest.webmanifest` em cada compilação do site.

### Usando `gatsby-plugin-manifest`

1.  Instale o plugin:

```shell
npm install --save gatsby-plugin-manifest
```

> 2. Adicione um favicon para o seu aplicativo em `src/images/icon.png`. O ícone é necessário para criar todas as imagens para o manifesto. Para mais informações, consulte os documentos em[`gatsby-plugin-manifest`](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-plugin-manifest/README.md).

Reviewer 3: ```suggestion
2. Adicione um favicon para o seu aplicativo em `src/images/icon.png`. O ícone é necessário para criar todas as imagens para o manifesto. Para mais informações, consulte a documentação disponível em[`gatsby-plugin-manifest`](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-plugin-manifest/README.md).
```

> title: Adding a Manifest File

Reviewer 4: ```suggestion
title: Adicionando o Arquivo _Manifest_
```

---

> Se você executou [audit with Lighthouse](/docs/audit-with-lighthouse/), você pode ter notado uma pontuação ruim na categoria "Progressive Web App". Vamos abordar como você pode melhorar essa pontuação.

Reviewer 5: ```suggestion
Se você executou uma [verificação com o Lighthouse](/docs/audit-with-lighthouse/), pode ter notado uma pontuação ruim na categoria "Progressive Web App". Vamos abordar como você pode melhorar essa pontuação.
```

Reviewer 5: Apenas para deixar um pouco mais claro ao usuário, veja o que acha desta melhoria.

Mas primeiro, o que _são_ exatamente PWAs?

> São sites regulares que aproveitam as funcionalidade modernas dos navegadores para aumentar a experiência na web com recursos e benefícios semelhantes a dos aplicativos. Confira [Google's overview](https://developers.google.com/web/progressive-web-apps/) para saber o que define uma experiência de PWA e [Progressive web apps (PWAs) doc](/docs/progressive-web-app/) para saber como um site Gatsby é um aplicativo da web progressivo.

Reviewer 4: ```suggestion
São sites regulares que aproveitam as funcionalidade modernas dos navegadores para aumentar a experiência na web com recursos e benefícios semelhantes a dos aplicativos. Confira [Google's overview](https://developers.google.com/web/progressive-web-apps/) para saber o que define uma experiência de PWA e [Progressive web apps (PWAs) doc](/docs/progressive-web-app/) para saber como um site Gatsby é um _progressive web app_.
```
Não sei se eu concordo em traduzir o "progressive web app". Tu tens alguma opinião sobre isso?

Reviewer 6: Concordo com o @jessescn quanto a não traduzir PWA.

> A inclusão de um manifesto de aplicativo da web é um dos três geralmente aceitos [baseline requirements for a PWA](https://alistapart.com/article/yes-that-web-project-should-be-a-pwa#section1).

Reviewer 4: ```suggestion
A inclusão de um manifesto de aplicativo da web é um dos três, geralmente aceitos, [requisitos mínimos para um PWA](https://alistapart.com/article/yes-that-web-project-should-be-a-pwa#section1).
```

Citação [Google](https://developers.google.com/web/fundamentals/web-app-manifest/):

> > O manifesto do aplicativo da web é um arquivo JSON simples que informa ao navegador sobre seu aplicativo da web e como ele deve se comportar quando 'instalado' no dispositivo móvel ou na área de trabalho do usuário.

Reviewer 6: ```suggestion
> O manifesto do aplicativo da web é um arquivo JSON simples que informa ao navegador sobre seu aplicativo web e como ele deve se comportar quando 'instalado' no dispositivo móvel ou na área de trabalho do usuário.
```

        start_url: "/",
        background_color: "#6b37bf",
        theme_color: "#6b37bf",

>         // Ativa o prompt "Adicionar à tela inicial" e desativa a interface do usuário do navegador (incluindo o botão Voltar)

Reviewer 3: ```suggestion
        // Ativa o prompt "Adicionar à tela inicial" e desativa os elementos de interface de usuário do navegador (incluindo o botão Voltar)
```

        // veja https://developers.google.com/web/fundamentals/web-app-manifest/#display
        display: "standalone",
        icon: "src/images/icon.png", // Esse caminho é relativo a raiz do site.
        // Um atributo opcional que fornece suporte para verificação CORS

>         // If you do not provide a crossOrigin option, it will skip CORS for manifest. Se você não fornecer a opção crossOrigin, ele pulará o CORS para manifesto

Reviewer 5: ```suggestion
        // Se você não fornecer a opção crossOrigin, ele pulará o CORS para o manifesto
```

Reviewer 5: Acho que faltou apagar aqui 😄

>         // Qualquer palavra-chave inválida ou string vazia assume o padrão "anônimo"

Reviewer 3: ```suggestion
        // Qualquer palavra-chave inválida ou string vazia usará por padrão o valor "anonymous"
```
